### DISCUSSION TITLE
Replication problem

### DISCUSSION THREAD

**Discussion by Basiasnow** [apache/couchdb#3516] [General]
Hi,

I would like repeat replication every one minute if something is wrong with replication first time.

Now, when I network off , on the next replication I have to wait exactly 4.6 minutes :

![image](https://user-images.githubusercontent.com/82805118/115206077-8515d200-a0fa-11eb-8a03-57bccdbc3262.png)

This can be problematic on the prod env.

I don't know what is wrong with this configuration :

![image](https://user-images.githubusercontent.com/82805118/115206130-9232c100-a0fa-11eb-9ece-170f73cd3ec7.png)

CouchDb ver. 3.1.1

**Comment by wohali** [comment]
One workaround is to restart CouchDB when the network comes back up. This would be essential if the IP address has changed (WiFi, mobile, etc.)
